orchestration/templates/cron-workflow.yaml

```
@@ -28,10 +28,22 @@ spec:
                 template: main
               arguments:
                 parameters:
+                  - name: data-repo-url
+                    value: {{ .Values.jade.url }}
+                  - name: dataset-id
+                    value: {{ .Values.jade.datasetId }}
+                  - name: jade-dataset-name
+                    value: {{ .Values.jade.datasetName }}
+                  - name: jade-data-project
+                    value: {{ .Values.jade.dataProject }}
+                  - name: jade-profile-id
+                    value: {{ .Values.jade.profileIddd }}
```

```suggestion
                    value: {{ .Values.jade.profileId }}
```
was this a typo too?